shared/model/loader.py

```diff
@@ -14,39 +14,54 @@
 
 import logging
 import os
+from typing import List
 
 from google.cloud import aiplatform
 from google.cloud import aiplatform_v1
 import yaml
 
 
-def _load_model_endpoints():
-  model_yaml_file = os.path.join(os.path.dirname(os.path.abspath(__file__)),
-                                 'vertex_ai_endpoints.yaml')
-  with open(model_yaml_file) as f:
+def _load_yaml(filename: str):
+  yaml_file = os.path.join(os.path.dirname(os.path.abspath(__file__)), filename)
+  with open(yaml_file) as f:
     return yaml.full_load(f)
 
 
-def load():
+def load_models(model_types: List[str] = None):
```

Comment:
nit: if the callers pass a single value, then avoid a list?

Reply:
I was thinking that eventually we may have more types and caller may want a list, but we can change that when the time comes, updated to single value